app/common/Dates.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017 HM Revenue & Customs
+ * Copyright 2018 HM Revenue & Customs
```

Comment:
In this file there is a reference to a localdate.now in the getCurrentYear method you might want to change.

Reply:
Good shout